src/test/decorators-spec.ts

```diff
@@ -0,0 +1,72 @@
+/* @license
+ * Copyright 2019 Google LLC. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {property, UpdatingElement} from 'lit-element';
+
+import {style} from '../decorators.js';
+import {numberNode} from '../styles/parsers.js';
+import {timePasses} from './helpers.js';
+
+const expect = chai.expect;
+
+const fooIntrinsics = {
+  basis: [numberNode(1, 'm'), numberNode(Math.PI, 'rad')],
+  keywords: {auto: [null, numberNode(200, '%')]}
+};
+
+class StyleableElement extends UpdatingElement {
+  @style({intrinsics: fooIntrinsics, updateHandler: 'updateFoo'})
+  @property({type: String})
+  foo: string = '200cm 1rad';
+
+  fooUpdates: Array<[number, number]> = [];
```

Comment:
Any reason for this to be an array and not just a couple of numbers that update?

Reply:
Just to assert that we don't invoke it more times than expected.